### PR TITLE
Username_userphoto

### PR DESCRIPTION
**Pull request by tomoneill32**
Posts now display who made the post, usernames and user ids are stored for each post in the database

### CODE REVIEW

File: cypress/integration/posts_display_who_made_the_post.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    cy.get("#password").type("password");
+    cy.get("#submit").click();
+
+    // submit a post
+    cy.visit("/posts");
+    cy.contains("New post").click();
+
+    cy.get("#new-post-form").find('[type="text"]').type("Hello, world!");
+    cy.get("#new-post-form").submit();
+
```
Comment (Rmorbey) [inline cypress/integration/posts_display_who_made_the_post.js]: Hey, should you also be testing to see if jerry smith's post is displayed while BoJo is logged in.

File: cypress/integration/user_can_see_username_on_timeline.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    cy.get('#username').type("Jane Doe");
+    cy.get("#submit").click();
+
+    // sign in
+    cy.visit("/sessions/new");
+    cy.get("#email").type("someone@gmail.com");
+    cy.get("#password").type("password");
+    cy.get("#submit").click();
+
+    cy.url().should("include", "/posts");
```
Comment (Rmorbey) [inline cypress/integration/user_can_see_username_on_timeline.js]: If we were in exampleserver:new/posts would that test pass also as they would both include /posts?

**Review by Rmorbey — COMMENTED** [review]
Good job guys, looks well tested and clean. Easy to read and understand.

File: cypress/integration/user_can_see_username_on_timeline.js (hunk 2 above)
Comment (tomoneill32) [reply cypress/integration/user_can_see_username_on_timeline.js]: This line is not necessary for this test, a duplicate from another test, removed

File: cypress/integration/posts_display_who_made_the_post.js (hunk 1 above)
Comment (tomoneill32) [reply cypress/integration/posts_display_who_made_the_post.js]: We have refactored the tests to one test, two users each make a post, the tests tests that the username is correct for each respective post

**Review by simon-2357 — APPROVED** [review]

**Pull request merged**